code-of-conduct.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at hi at opensourcediversity.org. All
```

Comment:
I think we should mention who is behind that email. Because for example, I (as a "reporter") wouldn't feel comfortable (or even not report the issue at all) reporting a CoC issue if the “the reported person” can read that mail. That's why I think it is very important we should say who reads/receive the mail and maybe near the names also their personal mail, or only one or two, in case the reported person can read/received emails that are sent at hi@...

Reply:
hi Jona, sorry for my late reply. I agree with what you've said and the changes you're proposing.

my full name is Charlie Kritschmar (which after an initial hesitation is okay to write here I think because it's pretty easy to connect it to Incabell with a quick Google search).

adding a few extra emails is probably a good idea as well. which ones should we take? I'm open to giving out mine but I'm also happy about anyone else who wants to volunteer theirs. 

thanks for setting all of this up!

Reply:
We could also set up redirects for at least the main recipients, like charlie at opensourcediversity.org, jona@, camila@ and so on. That way you wouldn’t need to give out your private email address and you can filter out. Does that seem good?

Reply:
Just went ahead and created them :slightly_smiling_face: we now have camila@, charlie@, greta@, jan@, jona@, kristi@ and nina@!

They forward to the mail which are also set as forwarders for the core@ address. Just send yourself an email to your new alias if you forgot which one that is. ;)

Not sure what the latest in spam prevention is, but we should probably also do something like writing it as e.g. `name at this domain dot org` because not everyone has (or should have ;) a Gmail account or a comparable spam filter.

Reply:
Oh and also – these are just the initial sets of addresses, we can of course add more people. We should probably discuss on the forum about what a "core" team means and there’s also no need to limit it to 7.